stories/components/TopBar.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import Col from 'react-bootstrap/lib/Col';
+import { storiesOf } from '@storybook/react';
+import StoryItem from '../styleguide/StoryItem';
+import TopBar from '../../lib/TopBar';
+import Button from '../../lib/Button';
+import Icon from '../../lib/Icon';
+import Avatar from '../../lib/Avatar';
+import AvatarGroup from '../../lib/AvatarGroup';
+
+storiesOf('Components', module).add('TopBar', () => (
+  <div>
+    <StoryItem
+      title="TopBar Wrapper"
+      description="TopBar is the top bar used throughout the whole site, can add a fixed attribute to fix it to the top."
+    >
+      <TopBar>
+        <Col className="top-bar__content top-bar__content--left" xs={10} md={5}>
+          <div className="top-bar__cell">
```

Comment:
To make the `Topbar` more descriptive and easier to digest do you think we should add components for certain children?

Two possible candidates would be `top-bar__content` & `top-bar__cell`. It has a number of modifiers that could be props too so it fits pretty well.

So it would be something like;

```
<TopBar>
  <TopBarContent left>
    <TopBarCell bordered>
      <Icon name="backArrow" className="top-bar__action top-bar__action--collapse-left" />
    </TopBarCell>
  <TopBarContent>
</TopBar>
```

You see where I'm going with this 😄